.changeset/real-dancers-search.md

```diff
@@ -0,0 +1,6 @@
+---
+'@remirror/core-utils': minor
+'@remirror/extension-link': minor
```

Comment:
```suggestion
```

This line isn’t needed since the changed files relate to the tests only.

Reply:
This PR does change behaviour of `commands.updateLink.isEnabled()` (only when the the link mark is not applicable) so could be relevant?

Reply:
That's a good point. 

Although, changes in `@remirror/extension-link` should already be captured by changesets in the individual package `CHANGELOG.md`.

However, I do see value in mentioning that the change provides a **known** bug fix to the link extension. I say, "known" because there are likely other places where the change has also had a positive impact. 

I'd prefer to capture that change in a separate `patch` changeset. This would mean that the changelog generated for the `@remirror/core-utils` package remains specific to what changed in the package.

The new changeset can be as follows:

```md
---
'@remirror/extension-link': patch
---

The behaviour of `commands.updateLink.isEnabled()` has been **fixed** to return `false` when the the `link` mark can't be applied to the selection. This was fixed by a change in the `@remirror/core-utils` package.
```

Reply:
Done :+1: